packages/components/src/notice/index.native.js

```diff
@@ -77,11 +77,27 @@ const Notice = ( { onNoticeHidden, content, id } ) => {
 		styles.noticeSolidDark
 	);
 
-	const textStyles = usePreferredColorSchemeStyle(
-		styles.text,
-		styles.textDark
+	const successTextStyles = usePreferredColorSchemeStyle(
+		styles.successText,
+		styles.successTextDark
 	);
 
+	const errorTextStyles = usePreferredColorSchemeStyle(
+		styles.errorText,
+		styles.errorTextDark
+	);
+
+	const getTextStyles = () => {
+		switch ( status ) {
+			case 'success':
+				return successTextStyles;
+			case 'error':
+				return errorTextStyles;
+			default:
+				throw new Error( `${ status } text style is not implemented.` );
+		}
+	};
```

Comment:
I was thinking that for only two cases maybe we can avoid using the switch approach? I'd also try to avoid using throwing an error and maybe just a warning instead (for development purposes).

We could also do it like this:

```suggestion
	const textStyles = [
		status === 'success' && successTextStyles,
		status === 'error' && errorTextStyles,
	];
```

What do you think?

Reply:
> I was thinking that for only two cases maybe we can avoid using the switch approach? I'd also try to avoid using throwing an error and maybe just a warning instead (for development purposes).

Understood. 

> What do you think?

Thanks for this suggestion @geriux . The logic is way more succinct now.  😄

Reply:
Resolved here 3785819